src/cli/cli.ts

```diff
@@ -1,7 +1,11 @@
+import { config } from 'dotenv';
 import yargs from 'yargs';
 import { hideBin } from 'yargs/helpers';
 import { ConstructAnalyzer } from '../lib/analyzer';
 
+// Load environment variables from .env file
+config();
```

Comment:
curious why you decided to go down this route. typically, we want to avoid adding dependencies wherever possible, and it seems simple enough to do this natively. you have an environment variable you want to use in your TS application, you can reference it via `process.env.MY_ENV_VAR`. Then it's on the developer to have `MY_ENV_VAR` set in their environment.